src/main/java/frc/robot/subsystems/swerve/SwerveModuleIoTalonFx.java

```diff
@@ -87,14 +87,17 @@ public SwerveModuleIoTalonFx(WheelModuleIndex index) {
 
     // Set Drive TalonFXConfiguration.
     var driveTalonFXConfiguration = new TalonFXConfiguration();
-    driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimit = 40.0;
-    driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimitEnable = true;
+    var currentLimits = driveTalonFXConfiguration.CurrentLimits;
```

Comment:
Should we move this entire object to Constants, so that we can use it on all TalonFx motors?

Reply:
Let's add a link to here for documentation reference: https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/CurrentLimitsConfigs.html

Reply:
We will likely want different current limits for different motors based on the expected load, but this would be a reasonable default.